src/user/supla_esp_gpio.h

```diff
@@ -136,13 +135,18 @@ void GPIO_ICACHE_FLASH supla_esp_gpio_state_update(void);
 
 void supla_esp_gpio_start_input_timer(supla_input_cfg_t *input_cfg);
 
-void supla_esp_gpio_set_hi(int port, char hi);
+void supla_esp_gpio_set_hi(int port, unsigned char hi);
```

Comment:
I'm not sure if switching to unsigned will break something somewhere.

Reply:
we are using 255 in code for "toggle". "char" may be signed or unsigned - this is not defined.
With "char" UT tests were not working properly with 255 value passed.